### ISSUE TITLE
Feature request: Add label "foo" when item moved into column "bar"

### ISSUE THREAD

**Issue by AndreasGalster** [philschatz/project-bot#70]
Right now we can move items when a label has been assigned. It's also useful to have the opposite workflow, so e. g. when I move something into the "QA" column, give the issue/PR a "in-qa" label.

Why? Our team uses the board to move items but we are using labels to filter specific issues by label (since projects don't offer this feature).

**Comment by jplehmann** [comment]
I agree, this is exactly what I was looking for.  Bidirectional sync between labels and project columns.

**Comment by danielytics** [comment]
I would also love this.
It would allow, for example, one project to control another to keep columns in sync: one project sets a label if the item is moved into a column, the second project moves the item into a column when the label is set. Now I can update one project and the other project is automatically kept in sync.

**Comment by panzacoder** [comment]
Adding in here that I would like this also. Our project planning is done through [Stories on Board](https://www.storiesonboard.com) where the GH issues integration is managed by way of labels. If I could have the doing and done labels automatically applied by moving to "In Progress" and "Done" that would be ideal.

**Comment by sherwa** [comment]
Is this done?
